examples/string-param/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"github.com/pellared/taskflow"
+)
+
+// Example program for parameter showcase.
+// This example also registers the "verbose" parameter, in order to provide output in the task.
+// Execute `go run ./main.go -v --example "hello world"` as an example.
+
+func main() {
+	flow := taskflow.New()
+
+	taskflow.VerboseParam(flow)
+	exampleParam := flow.ConfigureString("default-value", taskflow.ParameterInfo{
+		Name:  "example",
+		Short: 'e',
+		Usage: "An example parameter",
+	})
+
+	show := flow.MustRegister(taskShow(exampleParam))
```

Comment:
I am thinking if something like:
```go

show := taskShow(flow)

// ...

func taskShow(flow *taskflow.Taskflow) taskflow.RegisteredTask {
	exampleParam := flow.ConfigureString("default-value", taskflow.ParameterInfo{
		Name:  "example",
		Short: 'e',
		Usage: "An example parameter",
	})
	return flow.MustRegister(taskflow.Task{
		Name:        "show",
		Description: "Showcases a registered parameter",
		Parameters:  taskflow.Params{exampleParam},
		Command: func(tf *taskflow.TF) {
			tf.Log("Parameter named '" + exampleParam.Name() + "', value '" + exampleParam.Get(tf) + "'")
		},
	})
}
```

would be more readable if the parameter is used only in one task.
such structure would probably also visualize better what is shared between tasks

what do you think?

Reply:
I went further. I've extended the example to add a second task that shares the parameter with the first, plus uses a "private" parameter. This way multiple cases in one example.